src/base/simulation.jl

```diff
@@ -92,7 +91,7 @@ function _build_perturbations(perturbations::Vector{<:Perturbation})
     return callback_set, tstops
 end
 
-function _calculate_initial_conditions(sys::PSY.System, initial_guess::Vector{Float64})
+function _calculate_initial_conditions!(sys::PSY.System, initial_guess::Vector{Float64})
```

Comment:
We need to fill initial_guess with the result of NLsolve and just return the initialized flag. 


```suggestion
function _calculate_initial_conditions!(sys::PSY.System, initial_guess::Vector{Float64})
...
initial_guess = sys_solve.zero
return NLsolve.converged(sys_solve)
```

Reply:
Addressed via d80ff3b